backend/src/main/resources/graphql/main.graphqls

```diff
@@ -306,13 +307,19 @@ type Query {
   supportedDiseases: [SupportedDisease!]!
   specimenType: [SpecimenType]
   deviceSpecimenTypes: [DeviceSpecimenType]
+  facilities(
+    showArchived: Boolean = false
+      @requiredPermissions(allOf: ["VIEW_ARCHIVED_FACILITIES"])
```

Comment:
Will this block anyone without `VIEW_ARCHIVED_FACILITIES` permission from loading the facilities? or is it only applied to the parameter showArchived?

Reply:
It's applied to the parameter `showArchived`, and only blocks users without `VIEW_ARCHIVED_PERMISSIONS` from passing in anything other than `showArchived: false`.

This is based off of [this code comment](https://github.com/CDCgov/prime-simplereport/blob/8a0f6c4c2bbf881a7afefcd0cfdfd7b6b33b3b54/backend/src/main/resources/graphql/wiring.graphqls#L14) but I did also run into it by accident when I was trying to write auto for the query using the wrong user.